h2o-core/src/main/java/water/util/MathUtils.java

```diff
@@ -139,8 +142,11 @@ public void reduce(BasicStats bs) {
 
     private double[] variance(double[] res) {
       for (int i = 0; i < res.length; ++i) {
-        long nobs = _nobs - _naCnt[i];
-        res[i] = nobs == 0?0:(nobs / (nobs - 1.0)) * _m2[i] / _wsums[i];
+        //long nobs = _nobs - _naCnt[i];
+        //res[i] = nobs == 0?0:(nobs / (nobs - 1.0)) * _m2[i] / _wsums[i];
```

Comment:
The original calculation is too advanced for me to understand.  I just added my own calculation here.